doc/ref_arch/README.md

```diff
@@ -2,15 +2,38 @@
 
 # CNTT Reference Architectures
 
-<a name="1"></a>
+<a name="available-ra"></a>
 ## Available Reference Architectures
 * [Openstack Based](openstack)
 
-<a name="2"></a>
+<a name="principles"></a>
 ## Principles
-Those principles apply to all reference architecture produced by CNTT.
+**Preamble**
+
+CNTT develops a limited number of NFVI reference architectures. While technology and deployment aspects may differ between them, each of the CNTT reference architectures is based on a common CNTT Reference Model. 
+
+There are a number of key architectural principles that apply to all reference architectures produced by CNTT. These principles are enumerated below. They are meant to be general and at a high level, and limited in number. Some of the chapters of this document will include more specific principles to provide the implementation guidelines for a particular function or a specific component. Note that the Architectural Principles discussed in this Chapter follow the over-arching principles provided in [CNTT Reference Model:Introduction:Principles](https://cntt-n.github.io/CNTT/doc/ref_model/chapters/chapter01.html#1.3)
+
+
+We need to distinguish between architectural principles and architectural requirements elaborated later in this document. The principles are here to guide our architectural thinking, while requirements should be understood as a check list used to gauge a level of compliance of a NFVI implementation to the CNTT reference architecture.
+
+1. **Open source preference:** To ensure that suppliers’ and operators’ investment have a tangible pathway towards a standard and production ready NFVI solution portfolio. 
+
+1. **Open APIs:** To enable interoperability and component substitution, and minimize integration efforts.
+
+1. **Seperation of concerns:** To promote lifecycle indpendence of different architectural layers and modules (e.g. disagregation of software from hardware).
+
+1. **Automated lifecycle management:** To minimize costs of the end-to-end lifecycle, maintenance downtime (target zero downtime), avoid errors and discrepancies resulting from manual processes.
```

Comment:
This should be scoped only to the infrastructure itself.

Reply:
By default all principles apply to the CNTT scope as defined in CNTT RM

Reply:
Okay, scope is clear from [Chapter 1 of the RM](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter01.md#15scope), but there are usually misunderstantings about the scope. Okay to keep it like this now, but maybe we will need to be more clear if there are lost of misreads of this.